godot-cell/tests/mock.rs

```diff
@@ -433,5 +443,43 @@ fn calls_parallel_many_parallel() {
     unsafe {
         assert!(get_int(instance_id) >= min_expected);
         assert!(get_int(instance_id) <= max_expected);
+
+        #[cfg(feature = "threads")]
+        assert!(get_int(instance_id) == max_expected);
     }
 }
+
+/// Rwborrow the same cell on multible threads.
+///
+/// This verifies that threads don't hang if a reborrow occours inside of them, i.e. immutable borrow followed
+/// by a mutable borrow. Threads should only block if no local borrow exists but an other thread already has a
+/// mutable borrow or other threads already hold immutable borrows but the local thread requests it's first and only mutable borrow.
```

Comment:
```suggestion
/// Reborrow the same cell on multiple threads.
///
/// This verifies that threads don't hang if a reborrow occours inside them, i.e. immutable borrow followed
/// by a mutable borrow. Threads should only block if no local borrow exists but an other thread already has a
/// mutable borrow or other threads already hold immutable borrows but the local thread requests it's first and only mutable borrow.
```
The 2nd sentence "Threads should only..." is very hard to understand, could you maybe list up the conditions in bullet points? Also, please use commas between partial sentences 😊

Reply:
Additionally, it can make sense to write some of the comments inline inside the test, next to the thing it refers to.